forgerock-auth/src/main/java/org/forgerock/android/auth/SessionManager.java

```diff
@@ -44,8 +45,17 @@ public SessionManager(TokenManager tokenManager, SingleSignOnManager singleSignO
     }
 
     @WorkerThread
-    public void refresh(AccessToken accessToken, FRListener<AccessToken> listener) throws AuthenticationRequiredException {
-        tokenManager.refresh(accessToken, listener);
+    public void refresh(FRListener<AccessToken> listener) {
```

Comment:
Should we include javadoc here? I see more missing javadocs for other methods in this class

Reply:
Javadoc added.